apps/angular/permissions/src/app/user.store.ts

```diff
@@ -1,15 +1,29 @@
-import { Injectable } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
-import { User } from './user.model';
+import { computed, Injectable, signal } from '@angular/core';
+import { Role, User } from './user.model';
 
 @Injectable({
   providedIn: 'root',
 })
 export class UserStore {
-  private user = new BehaviorSubject<User | undefined>(undefined);
-  user$ = this.user.asObservable();
+  private _user = signal<User | undefined>(undefined);
```

Comment:
I think, if you are used to the _ naming convention for private field, you should switch to # which is a real javascript private field